scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-64_9x9.yaml

```diff
@@ -1,3 +1,54 @@
+QFN-64-1EP_9x9mm_P0.5mm_EP3.8x3.8xmm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://datasheet.lcsc.com/szlcsc/Realtek-Semicon-RTL8211EG-VB-CG_C69264.pdf (page 69)'
```

Comment:
Use the #page=xx style that we now use as it means the browser will directly open the correct page (also the pdf page should be used):

https://datasheet.lcsc.com/szlcsc/Realtek-Semicon-RTL8211EG-VB-CG_C69264.pdf#page=77